homeassistant/components/media_player/mpd.py

```diff
@@ -137,13 +139,20 @@ def name(self):
     def state(self):
         """Return the media state."""
         if self._status is None:
-            return STATE_OFF
+            self.update()
```

Comment:
This will cause network access in property

Reply:
Updated + corrected auto check things.